props/websubmit/src/main.rs

```diff
@@ -51,8 +59,13 @@ impl DeletionProp {
                     .collect::<Vec<_>>();
 
                 for t_src in &t_srcs {
-                    for delete in &delete_cs {
-                        if self.cx.data_flows_to(*c_id, t_src, delete) {
+                    for &delete in &delete_cs {
+                        if self.cx.flows_to(
+                            Some(*c_id),
+                            t_src,
+                            &CallSiteOrDataSink::DataSink(delete.clone()),
```

Comment:
Perhaps it would also make sense to provide a `From<CallSite>` and `From<DataSink>` implementation for `CallSiteOrDataSink` so we can just call `.into()` to crate it?

Reply:
Done